locallib.php

```diff
@@ -279,6 +289,24 @@ function local_bulkenrol_get_exception_info($e) {
     return " ".get_string('error_exception_info', 'local_bulkenrol').": ".$e->getMessage()." -> ".$e->getTraceAsString();
 }
 
+function create_users(&$localbulkenroldata) {
+    $userstocreate = $localbulkenroldata->users_to_be_created;
+    $emailsuffix = get_config('local_bulkenrol', 'email_suffix');
+    if (count($localbulkenroldata) > 0 && empty($emailsuffix)) {
```

Comment:
This causes an error, since the object in $localbulkenroldata is not countable. Why do you need this check anyways?

Reply:
Absolutely correct. It should count $userstocreate
The check is there to raise an error in case there is an empty email suffix